w3d_translator/classes/WandNavigation.py

```diff
@@ -0,0 +1,9 @@
+import xml.etree.ElementTree as ET
+from utils import tf_to_bool
+
+
+class WandNavigation:
```

Comment:
if all of your classes are going to have the `to_yaml` method, I'd consider adding a parent abstract class which declares that API (and maybe provides a default implementation if there turns out to be common ground)